app/domain/output_document.rb

```diff
@@ -20,13 +20,16 @@ def attendee_name
   end
 
   def attendee_address
+    attendee_country = Countries.uk?(self.attendee_country) ? nil : self.attendee_country
```

Comment:
Instead of doing this we could perhaps not delegate `attendee_country` to `appointment_summary` and instead add an `attendee_country` method to `OutputDocument` which encapsulates this logic